app/models/tests/test_test_case_model.py

```diff
@@ -98,6 +107,10 @@ def test_case_doc_to_dict_no_id(self):
         test_case.job = "this-job"
         test_case.kernel = "v123.45"
         test_case.lab_name = "area51"
+        test_case.log_lines = [{
+            'dt': datetime(1970, 1, 1, 0, 0, 0),
+            'msg': 'foo'}]
+        test_case.device_type = "pi"
```

Comment:
The `device_type` is already set a few lines above.

Reply:
corrected